weave/flow/scorers/llm_scorer.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any
+
+from pydantic import Field, field_validator
+
+from weave.flow.scorers.base_scorer import Scorer
+from weave.flow.scorers.llm_utils import _LLM_CLIENTS_NAMES, instructor_client
+
+
+class LLMScorer(Scorer):
+    """Score a model output using an LLM"""
+
+    client: Any = Field(
```

Comment:
so this is not really `Any` then?  Why not just pass the explicit list of types this can be?